package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "lab-07-cowsay-http-server",
+  "version": "0.0.1",
+  "description": "gnoevil lab-07",
+  "main": "server.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "nodemon server.js"
```

Comment:
Your start script should refer back to the node_modules folder in the same way Duncan referred to it in class. Refer to his lecture code and you can just copy/paste that script.